util/ubuntu-packages.txt

```diff
@@ -39,6 +39,7 @@ g++
 
 # C/C++ (and more) code-formatting tool
 clang-format
```

Comment:
Do we need this version of clang-format if we're forcing v10?

Reply:
We don't, I can remove this

Reply:
alright removed.